www/base/src/app/common/filters/publicFields.filter.coffee

```diff
@@ -0,0 +1,6 @@
+class PublicFields extends Filter('common')
+    constructor: ->
+        return (object) ->
+            for k of object
+                if k.indexOf('_') == 0 then delete object[k]
```

Comment:
delete?
Are you modifying the original object? sounds bad.
why dont you rather build a copy